70_FLIPFluids/app_resources/fluidParticles.fragment.hlsl

```diff
@@ -0,0 +1,35 @@
+#pragma shader_stage(fragment)
+
+#include "common.hlsl"
+#include "render_common.hlsl"
+
+[[vk::binding(0, 1)]]
+cbuffer CameraData
+{
+    SMVPParams camParams;
+};
```

Comment:
UBOs don't really give advantages over SSBO on desktop GPUs and we use BDA instead of SSBO now.

This is far less painful to manage because you don't need to:
1. declare a UBO slot in the descriptor layout
2. write a descriptor set
3. do all the managment

you just pass a device address as a `uint64_t` in push constants and read from it like a casted `void*` using `vk::RawBufferLoad`